### DISCUSSION TITLE
How do I get FluentProfileMenu to work with Identity Pages like login, register etc

### DISCUSSION THREAD

**Discussion by coderdnewbie** [microsoft/fluentui-blazor#2423] [Q&A]
I am having a strange issue, where I can not use the Identity pages and the FluentProfileMenu popover at the same time.

For the identity pages to work App.Razor needs to have Routes set without any mode set, i.e. `<Routes />`.

However, for the FluentProfileMenu popover to be displayed (from the MainLayout.razor), the mode must be set in the Routes. Otherwise the Profile card popup is not displayed, i.e. it needs to be `<Routes @rendermode="InteractiveServer"/>`.

So it appears I cannot have both features working at the same time. Has anyone managed to make this work, so that I can use the identity pages and also am able to see the FluentProfileMenu Popover Card?

Thanks.

**Comment by vnbaaij** [comment]
You should be able to just set the `@rendermode="InteractiveServer"` as an attribute in the ProfileMenu call in MainLayout.

**Reply by prod42net** [reply]
I did this. It is getting displayed but I, for some reason, get the OnHeaderButtonClick ( Sign out ) as well as the OnFooterLinkClick not to work. Both are showing up ...but it will never call any method I assign to it. Where is the issue with my thinking?

**Reply by vnbaaij** [reply]
The account pages are rendered in SSR mode. I'm assuming your ProfileMenu is using SSR as well, so there is no interactivity and buttons won't do anything.

**Comment by coderdnewbie** [comment]
Thanks, I did not know I could do that. It worked.
